components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/AuthorizationParams.kt

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.appservices.fxaclient
+
+data class AuthorizationParams(
+    val clientId: String,
+    val scopes: Array<String>,
+    val state: String,
+    val accessType: String = "online",
+    val keyParams: AuthorizationKeyParams? = null
+) {
+    fun intoMessage(): MsgTypes.AuthorizationParams {
+        var b = MsgTypes.AuthorizationParams.newBuilder().setClientId(this.clientId)
```

Comment:
nano-nit: I almost missed the call to `setClientId` here, I wonder if it's worth putting it on a newline for easier visibility (unless this is auto-formatted for kotlin guidelines, in which case no big deal).